docs/source/usage.rst

```diff
@@ -213,7 +213,7 @@ PS: This feature needs the email settings to be configured similar to `4. Allow
 7. Enable blocking system
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Blocking functionality is added in version 2.7.0. It allows moderators to block users/emails from creating/editing or reacting to a comment.
+Blocking functionality is added in version `2.7.0`. It allows moderators to block users/emails from creating/editing or reacting to a comment.
```

Comment:
This change is not necessarily related to the release. I would make this in a separate commit with something like `Correct Typo in ...`.

Reply:
you may still make the commit in this pull-request itself.